node/pkg/driver/node_utils.go

```diff
@@ -166,3 +169,50 @@ func (n NodeUtils) StageInfoFileIsExist(filePath string) bool {
 	}
 	return true
 }
+
+func (n NodeUtils) ParseFCPortsName(path string) ([]string, error) {
+	//FC port path is /sys/class/fc_host/host*/port_name
+	var errStrings []string
+	var FCPorts []string
+
+	fpaths, err := n.Executer.FilepathGlob(path)
+	if fpaths == nil {
+		err = fmt.Errorf(ErrorUnsupportedConnectivityType, "FC")
+	}
+	if err != nil {
+		return nil, err
+	}
+
+	for _, fpath := range fpaths {
+		file, err := os.Open(fpath)
+		if err != nil {
+			errStrings = append(errStrings, err.Error())
+			break
+		}
+		defer file.Close()
+
+		file_out, err := ioutil.ReadAll(file)
+		if err != nil {
+			errStrings = append(errStrings, err.Error())
+			break
+		}
+
+		fileSplit := strings.Split(string(file_out), "0x")
```

Comment:
you can use strings.Replace(string(file_out), "0x", "", -1), and don't forget 0X too